### ISSUE TITLE
How to parse a Json Log

### ISSUE THREAD

**Issue by dtavolato** [sergey-su/logjoint#42]
Hello Guys and thank for the great job.
I would like to use LogJoint to parse and merge multiple log file, those log are im Json format. 
Is it possible to parse like LogJoint already does with XML log?

**Comment by sergey-su** [comment]
Yes, it's possible to parse JSON logs. Your JSON format has to be described first, same as with XML logs. If you select Log Sources -> Add... -> Manage Formats..., you can open a wizard that can help describe the format. In the wizard, there is a help article.

If you have difficulties with the wizard, you can send a sample of your json log, and I can help create the format description.

**Comment by dtavolato** [comment]
Hello! Thanks for your feedback. Here is a sample of my log format. It would be nice if you could help me with this.
Thanks!
[logFile_INFO.20241120.1.json](https://github.com/user-attachments/files/17929801/logFile_INFO.20241120.1.json)

**Comment by sergey-su** [comment]
Here you go.

Unzip the XML format description file into C:\Users\your_user\AppData\Local\LogJoint\bin\Formats or whatever your installation dir is. Then LogJoint should recognize the log format once you drag&drop a log.

You may want to edit the XML. I assumed that errors and warnings are encoded as `"level": "ERROR"` and `"level": "WARN"` respectively. I also assumed that encoding is utf-8 and the file extension is *.json.

Your sample log is not strictly ordered by time, so I added an option to dynamically reorder the logs with window size of 50 messages.

[dtavolato - JSON log.format.zip](https://github.com/user-attachments/files/17939780/dtavolato.-.JSON.log.format.zip)
